elements/cloud-init-growpart-lvm/post-install.d/55-growpart-lvm

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ ${DIB_DEBUG_TRACE:-1} -gt 0 ]; then
+    set -x
+fi
+set -euo pipefail
+
+if [[ -n ${DIB_CLOUD_INIT_GROWPART_DEVICES} ]]; then
+    if [ -d /etc/cloud/cloud.cfg.d ]; then
+        cat > /etc/cloud/cloud.cfg.d/55-growpart-lvm.cfg <<EOF
+    #cloud-config
+    growpart:
+      mode: auto
+      devices: $DIB_CLOUD_INIT_GROWPART_DEVICES
+      ignore_growroot_disabled: false
+    EOF
```

Comment:
I think EOF needs to be at the start of the line

Reply:
In fact everything from #cloud-config needs to be dedented.

Reply:
done